release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1341,12 +1341,14 @@ For any {product-title} release, always review the instructions on xref:../updat
 ====
 
 [id="ocp-4-7-0-ga"]
-=== RHBA-2021:1234 - {product-title} 4.7 image release and bug fix advisory
+=== RHEA-2020:5633 - {product-title} 4.7.0 image release, bug fix, and security update advisory
 
-Issued: 2021-xx-xx
+Issued: 2021-02-24
 
-{product-title} release 4.7 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2020:1234[RHBA-2020:1234] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2020:5678[RHBA-2020:5678] advisory.
+{product-title} release 4.7.0 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHEA-2020:5633[RHEA-2020:5633] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHEA-2020:5634[RHEA-2020:5634] advisory.
```

Comment:
I am assuming that these links are not live until GA. They return a 404 for me at present.

Reply:
You could swap the last two sentences of this paragraph around, to introduce the RPM packages before the bugs. This would introduce the product update first and then resolutions to issues afterwards. For example:

"{product-title} release 4.7.0 is now available. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHEA-2020:5634[RHEA-2020:5634] advisory. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHEA-2020:5633[RHEA-2020:5633] advisory."

Reply:
You could minimise the bug fix related sentence in one of the two following ways:

"The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHEA-2020:5633[RHEA-2020:5633] advisory."

"The list of bug fixes that are included in the update is in the link:https://access.redhat.com/errata/RHEA-2020:5633[RHEA-2020:5633] advisory."

Reply:
Point well taken. Don't want to throw L10N a curveball, but will revisit post-GA.